netbench-orchestrator/src/orchestrator/cli.rs

```diff
@@ -0,0 +1,121 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::orchestrator::{OrchError, OrchResult};
+use serde::Deserialize;
+use std::{
+    fs::File,
+    path::{Path, PathBuf},
+};
+
+#[derive(Clone, Debug)]
+pub struct OrchestratorConfig {
+    // netbench
+    netbench_scenario_filename: String,
+    netbench_scenario_filepath: PathBuf,
+
+    // cdk
+    pub cdk_config: CdkConfig,
+}
+
+// Used for parsing the scenario file generated by the s2n-netbench project
+#[derive(Clone, Debug, Default, Deserialize)]
+pub struct CdkConfig {
+    #[serde(rename(deserialize = "NetbenchInfraPrimaryProd"))]
+    resources: CdkResources,
+}
+
+#[derive(Clone, Debug, Default, Deserialize)]
+#[serde(rename_all(deserialize = "camelCase"))]
+struct CdkResources {
+    output_netbench_runner_log_group: String,
+    output_netbench_runner_public_logs_bucket: String,
+    output_netbench_runner_private_src_bucket: String,
+    output_netbench_cloudfront_distribution: String,
+    output_netbench_runner_instance_profile: String,
+    output_netbench_subnet_tag_key: String,
+    output_netbench_subnet_tag_value: String,
+    output_netbench_infra_primary_prod_region: String,
```

Comment:
`primary_prod_region` implies the existence of a secondary prod region? Maybe "deployment region" or something like that?

Reply:
Perhaps changing the name is out of scope, but could we just add a comment clarifying what exactly this region is or is not used for?

Reply:
Oh, also there might be secondary regions in the future if we support multi-regions